pkgs/test/coq/overrideCoqDerivation/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, coq, mkCoqPackages, runCommandNoCC }:
+
+let
+
+  # This is just coq, but with dontFilter set to true. We need to set
+  # dontFilter to true here so that _all_ packages are visibile in coqPackages.
+  # There may be some versions of the top-level coq and coqPackages that don't
+  # build QuickChick, which is what we are using for this test below.
+  coqWithAllPackages = coq // { dontFilter = true; };
+
+  coqPackages = mkCoqPackages coqWithAllPackages;
+
+  # This is the main test.  This uses overrideCoqDerivation to
+  # override arguments to mkCoqDerivation.
+  #
+  # Here, we override the defaultVersion and release arguments to
+  # mkCoqDerivation.
+  overriddenQuickChick =
+    coqPackages.lib.overrideCoqDerivation
+      {
+        defaultVersion = "9999";
+        release."9999".sha256 = lib.fakeSha256;
+      }
+      coqPackages.QuickChick;
+in
+
+runCommandNoCC
```

Comment:
```suggestion
runCommand
```

Reply:
What's the reasoning for preferring `runCommand` instead of `runCommandNoCC`?  I was under the impression that `runCommandNoCC` should be used over `runCommand` because `runCommandNoCC` doesn't pull in a whole C toolchain, thus making the build-time dependencies for the derivation a little smaller.

Reply:
Oh, except now that I looked it up, I can see that `runCommandNoCC` is now just an alias for `runCommand`!  Didn't realize this change happened.  Thanks for pointing it out!

Fixed in